compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1038,6 +1038,15 @@ error.duplicate.key.in.table.literal=\
 error.duplicate.key.in.record.literal.spread.op=\
   invalid usage of {0} literal: duplicate key ''{1}'' via spread operator ''{2}''
 
+error.possible.duplicate.of.field.specified.via.spread.op=\
+  invalid usage of mapping constructor expression: key ''{0}'' may duplicate a key specified via spread field ''{1}''
+
+error.already.specified.keys.in.spread.field=\
```

Shall we rename this and the DiagnosticCode to reflect that it is a possible duplicate?

e.g.,
```suggestion
error.spread.field.may.duplicate.already.specified.keys=\
```